ipyparallel/engine/engine.py

```diff
@@ -66,7 +67,23 @@ def tunnel_mod(self):
     # not configurable:
     connection_info = Dict()
     user_ns = Dict()
-    id = Integer(allow_none=True)
+    id = Integer(None, allow_none=True, config=True,
+        help="""Request this engine ID.
+        
+        If run in MPI, will use the MPI rank.
+        Otherwise, let the Hub decide what our rank should be.
+        """
+    )
+    @default('id')
+    def _id_default(self):
+        try:
+            from mpi4py import MPI
```

Comment:
Is it possible to move this to a MPI specific engine? the existence of mpi4py package doesn't suggest the user wants to initialize MPI.